### PR TITLE
Issue/select

### PR DESCRIPTION
**Pull request by BelohvostovAlex**

### CODE REVIEW

File: src/components/Todo/Todo.tsx [hunk 1]
```diff
@@ -11,24 +11,43 @@ export const Todo: React.FC<TodoProps> = ({
...
+          <RoundButton
+            btnText="X"
+            btnClass="round-btn round-btn--red"
+            onClickBtn={() => deleteTodo(id)}
+          />
+          <RoundButton
+            btnText="E"
+            btnClass="round-btn round-btn--yellow"
+            onClickBtn={() => handleEditVisibleModal(id)}
+          />
```
Comment (igorTT) [inline src/components/Todo/Todo.tsx:49]: There is logic from another ticket. Wait until it'll be merged or delete it

**Review by igorTT — CHANGES_REQUESTED** [review]

File: src/components/Todo/Todo.tsx (hunk 1 above)
Comment (BelohvostovAlex) [reply src/components/Todo/Todo.tsx]: deleted

File: src/components/Todo/todo.scss [hunk 2]
```diff
@@ -25,6 +25,39 @@
...
   display: flex;
   flex-direction: column;
-  align-items: center;
+  align-items: flex-start;
   justify-content: center;
+}
+
+.todo-select {
+  width: 150px;
+  max-width: 200px;
```
Comment (igorTT) [inline src/components/Todo/todo.scss:34]: Why do we need the max-width?

File: src/components/Todo/todo.scss [hunk 3]
```diff
@@ -25,6 +25,39 @@
...
+  padding: 7px 20px;
+  text-align: center;
+  -moz-appearance: none;
+	-webkit-appearance: none;
+  border: 1px solid black;
+  border-radius: 5px;
```
Comment (igorTT) [inline src/components/Todo/todo.scss:40]: I don't see the rounded borders on the mockup

**Review by igorTT — CHANGES_REQUESTED** [review]
image
Looks like a problem

File: src/components/Todo/todo.scss (hunk 2 above)
Comment (BelohvostovAlex) [reply src/components/Todo/todo.scss:34]: deleted

File: src/components/Todo/todo.scss (hunk 3 above)
Comment (BelohvostovAlex) [reply src/components/Todo/todo.scss:40]: deleted

File: src/components/Todo/Todo.tsx [hunk 4]
```diff
@@ -6,23 +6,41 @@ import { RoundButton } from '../RoundButton/RoundButton';
...
+          onChange={handleOption}
+          className={classes}
+          value={selectedOption}
+        >
+          {avaliableOptions.map((option) => (
+            <option key={option} value={option}>
+              {option}
+            </option>
+          ))}
+        </select>
```
Comment (igorTT) [inline src/components/Todo/Todo.tsx:36]: Move it to the separate component

File: src/components/Todo/Todo.tsx [hunk 5]
```diff
@@ -6,23 +6,41 @@ import { RoundButton } from '../RoundButton/RoundButton';
...
+        <div className="todo-buttons">
```
Comment (igorTT) [inline src/components/Todo/Todo.tsx:37]: Useless wrapper

File: src/components/Todo/todo.scss [hunk 6]
```diff
@@ -27,6 +27,38 @@
...
   flex-direction: column;
-  align-items: center;
+  align-items: flex-start;
   justify-content: center;
+}
+
+.todo-select {
+  width: 150px;
+  height: 40px;
+  padding: 7px 20px;
```
Comment (igorTT) [inline src/components/Todo/todo.scss:37]: Useless padding

**Review by igorTT — CHANGES_REQUESTED** [review]

File: src/components/Todo/Todo.tsx (hunk 5 above)
Comment (BelohvostovAlex) [reply src/components/Todo/Todo.tsx:37]: removed

File: src/components/Todo/todo.scss (hunk 6 above)
Comment (BelohvostovAlex) [reply src/components/Todo/todo.scss:37]: removed

File: src/components/Todo/Todo.tsx (hunk 4 above)
Comment (BelohvostovAlex) [reply src/components/Todo/Todo.tsx:36]: done

File: src/components/Todo/TodoContainer.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  id,
+  title,
+  description,
+  deleteTodo,
+}) => {
+  const [avaliableOptions, setAvaliableOptions] = useState([
+    'Todo',
+    'In progress',
+    'Done',
+  ]);
```
Comment (igorTT) [inline src/components/Todo/TodoContainer.tsx:17]: What does this state do?
Config eslint on your code editor. It should highlight this kind of errors

**Review by igorTT — CHANGES_REQUESTED** [review]

File: src/components/Todo/TodoContainer.tsx (hunk 7 above)
Comment (BelohvostovAlex) [reply src/components/Todo/TodoContainer.tsx:17]: removed state,
yes, i had a warning, thought warnings = okay, no problem, just warning)

File: src/components/Todo/TodoContainer.tsx [hunk 8]
```diff
@@ -10,11 +10,7 @@ export const TodoContainer: React.FC<TodoContainerProps> = ({
   description,
   deleteTodo,
 }) => {
-  const [avaliableOptions, setAvaliableOptions] = useState([
-    'Todo',
-    'In progress',
-    'Done',
-  ]);
+  const availiableOptions = ['Todo', 'In progress', 'Done'];
```
Comment (igorTT) [inline src/components/Todo/TodoContainer.tsx:13]: Store this array out of the component. It'll help with memoization.

**Review by igorTT — CHANGES_REQUESTED** [review]

File: src/components/Todo/TodoContainer.tsx (hunk 8 above)
Comment (BelohvostovAlex) [reply src/components/Todo/TodoContainer.tsx]: Should i store it outside the func body?

**Review by igorTT — APPROVED** [review]

**Comment by igorTT** [comment]
@BelohvostovAlex , wait for https://github.com/BelohvostovAlex/todo/tree/issue/base will be merged

**Review by igorTT — APPROVED** [review]

**Pull request merged**